src/AsyncTCP.h

```diff
@@ -29,9 +33,11 @@ extern "C" {
 #else
 extern "C" {
 #include <lwip/pbuf.h>
+#include <FreeRTOS.h>
 #include <semphr.h>
 }
 #define CONFIG_ASYNC_TCP_RUNNING_CORE -1  // any available core
+#define CONFIG_FREERTOS_UNICORE 1
```

Comment:
same question here: should this macro be defined elsewhere ? Seems to me that this should be a  global flag ?

Reply:
Maybe it is better to check for libretiny where this macro is used and not define it ?